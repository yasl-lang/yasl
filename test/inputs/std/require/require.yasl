##10\nG: 10\n10\nG: 11\n11\n11\n11\n3\nsubmodule\n11\nG: 11\n11\nG: 12\n12\n12\n12\n10\nG: 10\n10\nG: 11\n11\n11\n11\n20\n

let xyz = 20
const s = require('test/inputs/submodule.yam')

echo s.g()
echo s.f()
echo s.g()

echo s.add(1, 2)
echo s.sub
echo s.g()
echo s.f()
echo s.g()

let s2 = require('test/inputs/submodule.yam')

echo s2.g()
echo s2.f()
echo s2.g()

echo xyz
