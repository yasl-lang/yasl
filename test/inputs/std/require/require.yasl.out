10
G: 10
10
G: 11
11
11
11
3
submodule
11
G: 11
11
G: 12
12
12
12
10
G: 10
10
G: 11
11
11
11
20
